src/extensions/export/bootstrap-table-export.js

```diff
@@ -23,8 +24,7 @@
     $.extend($.fn.bootstrapTable.defaults, {
         showExport: false,
         exportDataType: 'basic', // basic, all, selected
-        // 'json', 'xml', 'png', 'csv', 'txt', 'sql', 'doc', 'excel', 'powerpoint', 'pdf'
-        exportTypes: ['json', 'xml', 'csv', 'txt', 'sql', 'excel'],
+        exportTypes: ['json', 'html', 'xml', 'csv', 'txt', 'sql', 'pdf', 'excel', 'doc', 'powerpoint'],
```

Comment:
@hhurz thank you again for your notes I apperciate very much. While wenzhixin take a decision, i continue in my own fork (must to add txt and png export and some options).

For the difference between actual export options and my export options, i can write a "convert options" while users edit their code.